medusa/show_queue.py

```diff
@@ -397,30 +408,32 @@ def run(self):
                 (self.showDir, indexerApi(self.indexer).name, self.indexer_id)
             )
 
-            if app.USE_TRAKT:
-
-                trakt_id = indexerApi(self.indexer).config['trakt_id']
-                trakt_api = TraktApi(app.SSL_VERIFY, app.TRAKT_TIMEOUT)
-
-                title = self.showDir.split("/")[-1]
-                data = {
-                    'shows': [
-                        {
-                            'title': title,
-                            'ids': {}
-                        }
-                    ]
-                }
-                if trakt_id == 'tvdb_id':
-                    data['shows'][0]['ids']['tvdb'] = self.indexer_id
-                else:
-                    data['shows'][0]['ids']['tvrage'] = self.indexer_id
-
-                try:
-                    trakt_api.traktRequest("sync/watchlist/remove", data, method='POST')
-                except TraktException as e:
-                    logger.log(u"Could not remove show '{0}' from watchlist. Error: {1}".format
-                               (title, e), logger.WARNING)
+            # FIXME: This shouldn't be here. I've commented it. If nobody can provide me with reasons to keep it.
```

Comment:
@p0psicles i used that so im going to explain.

I add a show 1) in trakt watchlist, medusa syncs and add show or 2) in medusa and show is added to trakt watchlist
I watch first episode but didn't like. then i remove show. then medusa removes this show from trakt

if you think it shouldn't be there, at least the call to the function that removes should be.